src/screens/MealList/mealCard/MealCard.tsx

```diff
@@ -1,5 +1,5 @@
-import { Text, Card, CardItem, View } from 'native-base';
-import { Image } from 'react-native';
+import { Text, Card, CardItem } from 'native-base';
+import { Image, StyleSheet, View } from 'react-native';
```

Comment:
No need to import StyleSheet here. Next time we meet, we can set up a linter for you to catch these small things